kalite/distributed/static/js/distributed/exercises.js

```diff
@@ -1413,17 +1413,6 @@ window.ExerciseQuizView = Backbone.View.extend({
 
         if(this.log_model.get("attempts")==1){
             if(this.points > 0){
-                var purchased_model = new PurchasedStoreItemModel({
-                    item: "/api/store/storeitem/gift_card/",
-                    purchased_at: window.statusModel.get_server_time(),
-                    reversible: false,
-                    context_id: ds.ab_testing.unit || 0,
-                    context_type: "unit",
-                    user: window.statusModel.get("user_uri"),
-                    value: this.points
-                });
-                purchased_model.save();
-
```

Comment:
Stuck here, please suggest whether I implement points for Quiz and if yes then how do I do it in QuizLog model, similar to exercises?